### DISCUSSION TITLE
Comet Kiwi training data

### DISCUSSION THREAD

**Discussion by loic-mt** [eole-nlp/eole#127] [General]
Hello!

Looking at the Comet Kiwi recipe, I see the model gets fine tuned on a corpus named '1720-da.mlqe'.
Please does someone here know what this corpus is and where to find it? Maybe @vince62s ?

I looked into both Unbabel and MLQE-PE repos without success.

Thank you!

Loic

**Comment by vince62s** [comment]
Depend on what you are trying to achieve.
Raw data is here: https://github.com/Unbabel/COMET/issues/208

But it requires a bit of preprocessing if you want to run the recipe.

**Comment by vince62s** [comment]
if you want the model and data used to train the model here you go: https://huggingface.co/eole-nlp/cometkiwi-xxl-eole/tree/main
